### PR TITLE
export branch stats api endpoint doc

### PR DESCRIPTION
**Pull request by bjornj12**
simple docs for branch stats endpoint

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…pi/export-branch-stats**](https://docs-q63dqbfct.avo-dev.app/reference/public-api/export-branch-stats) |
|:-:|
|<a href="https://docs-q63dqbfct.avo-dev.app/reference/public-api/export-branch-stats">Screenshot of /reference/public-api/export-branch-stats</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/HUvRwbbCc2nxE6l07qIYhnt9LN7CXyx1bc5Oh8Kz3q_1iB_jnAHWeNHisgXdevVkEwNldM4I563zRjo2k45hJw.png">(view full size)</a> |







Commit 833f9f2bdec753ae4318ca51eae72efc570ad3ec (https://docs-q63dqbfct.avo-dev.app).

File: pages/reference/public-api/export-branch-stats.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+{/* cSpell:disable */}
```
Comment (logason) [inline pages/reference/public-api/export-branch-stats.mdx:1]: We want this spellchecked, right 👀

File: pages/reference/public-api/export-branch-stats.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+## Endpoint
+
+```Url
+https://api.avo.app/workspaces/:workspaceId/branches/export/v1
+```
+
+`:workspaceId` is the ID of your workspace. You'll find it in the URL of your avo tab after `/schemas/`.
+
+Returns [a CSV string that is a representation of your branch stats]
```
Comment (logason) [inline pages/reference/public-api/export-branch-stats.mdx:17]: Which branches are included in the response? All branches in history, only open ones, or something else?

File: pages/reference/public-api/export-branch-stats.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+- `branchName`: Name of the branch
+- `branchId`: Id of the branch
+- `branchCreatorId`: userId of the branch creator
+- `branchCreatorEmail`: email of the branch creator
+- `creationDate`: UTC timestamp when the branch was created
+- `branchStatus`: current status on the branch [Merged | Closed | Approved | Draft | ReadyForReview]
+- `collaborators`: array of user emails that helped on the branch.
+- `lastPublishedDate`: UTC timestamp of last time when the branch was published to a downstream publishing integration
+- `markedReadyForReviewDate`: UTC timestamp when the branch was marked as ReadyForReview
```
Comment (logason) [inline pages/reference/public-api/export-branch-stats.mdx:29]: I assume this is the for the first time a branch is marked as ready for review, for cases where that is done multiple times on the same branch?

File: pages/reference/public-api/export-branch-stats.mdx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+- `markedReadyForReviewUser`: email of the user who marked the branch as ReadyForReview
```
Comment (logason) [inline pages/reference/public-api/export-branch-stats.mdx:30]: To be consistent with `branchCreatorEmail`
```suggestion
- `markedReadyForReviewerUserEmail`: email of the user who marked the branch as ReadyForReview
```

File: pages/reference/public-api/export-branch-stats.mdx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+- `branchMergedDate`: UTC timestamp when the branch was merged
+- `branchMergedUser`: email of the user who merged the branch.
```
Comment (logason) [inline pages/reference/public-api/export-branch-stats.mdx:32]:
```suggestion
- `branchMergedUserEmail`: email of the user who merged the branch.
```

File: pages/reference/public-api/export-branch-stats.mdx (hunk 1 above)
Comment (logason) [inline pages/reference/public-api/export-branch-stats.mdx:1]: For some reason I can't find this page in the Vercel build 🤔

File: pages/reference/public-api/export-branch-stats.mdx (hunk 3 above)
Comment (bjornj12) [reply pages/reference/public-api/export-branch-stats.mdx:29]: yes

File: pages/reference/public-api/overview.mdx [hunk 6]
```diff
@@ -6,3 +6,4 @@ Avo API returns JSON encoded responses and uses standard HTTP response codes.
 Avo API Uses [Basic Authentication](https://en.wikipedia.org/wiki/Basic_access_authentication) to authenticate requests.
 
 - [Export Tracking Plan](/public-api/export-tracking-plan): Export your Avo tracking plan as JSON from any branch
+- [Export Branch Stats](/public-api/export-branch-stats): Export your Avo branch stats as CSV for all branches in the workspace
```
Comment (logason) [inline pages/reference/public-api/overview.mdx:9]: I wonder if we should just call it Branches? (if you agree, let's change the header in `/export-branch-stats.mdx` as well)
```suggestion
- [Export Branches](/public-api/export-branch-stats): Export your Avo branch stats as CSV for all branches in the workspace
```

**Review by logason — APPROVED** [review]

**Pull request merged**